tools/kubernetes/helm/hue/templates/prometheus-alert-rule.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.alertrule.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: example
```

Comment:
Is this one needed?

Reply:
If we are speaking of **prometheus: example** ,This label is required without which alert shall not reflect in alert section of prometheus. 
Also I suggest to keep also following up with reference to [doc](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/alerting.md)

_The best practice is to label the PrometheusRules containing rule files with role: alert-rules as well as the name of the Prometheus object, prometheus: example in this case._ 

Please suggest further . Thanks

Reply:
I see, and thanks for the link! So just how about a better name like `main` and make it a variable too? (people might have different names)

Reply:
True, Let me check the right labelling for this and update this section.

Reply:
I tried to understand here a bit more about the _prometheus_ label and understood that it should match with the prometheus pods that might end up running in monitoring/other namespace. I could define a variable but then i felt it would be better to name it as hue considering people might not have this value defined to be picked from helm/release etc , Thus update this to _promethues : hue_  following all other labeling that we have (service etc) . 
Felt hue label could be better and easy to compare if people are missing something than putting it like a _promethues : main_ .  

I can define this also in values.yaml if required and later pick this up {{ .Values.alerting.promethus-label }} do let me know which works out best.